extensions/ql-vscode/package.json

```diff
@@ -197,6 +197,10 @@
           "dark": "media/dark/folder-opened-plus.svg"
         }
       },
+      {
+        "command": "codeQLDatabases.removeOrphanedDatabases",
+        "title": "Remove databases no longer imported into VS Code"
```

Comment:
Clean up unused databases?

Reply:
This is not really a user facing command. But, sure.